pkg/services/librarypanels/models.go

```diff
@@ -22,8 +22,10 @@ type LibraryPanel struct {
 }
 
 var (
-	// errLibraryPanelAlreadyAdded is an error when you add a library panel that already exists.
-	errLibraryPanelAlreadyAdded = fmt.Errorf("library panel with that title already exists")
+	// errLibraryPanelAlreadyAdded is an error for when the user tries to add a library panel that already exists.
+	errLibraryPanelAlreadyAdded = errors.New("library panel with that title already exists")
+	// errLibraryPanelNotFound is an error for when a library panel can't be found.
+	errLibraryPanelNotFound = errors.New("library panel could not be found")
```

Comment:
Changed to errors.New but not sure that is more correct.

Reply:
Since you don't need any special format, I think `errors.New()` it's fine.
`fmt.Errorf()` allows to format the error message using variables or wrap another error.

Reply:
Aha, so that's the use case. Then this is better right now, thank you for clarifying it.